scylla/src/transport/load_balancing/mod.rs

```diff
@@ -41,16 +41,17 @@ pub struct RoutingInfo<'a> {
 /// (or when speculative execution is triggered).
 pub type FallbackPlan<'a> = Box<dyn Iterator<Item = (NodeRef<'a>, Shard)> + Send + Sync + 'a>;
 
-/// Policy that decides which nodes to contact for each query.
+/// Policy that decides which nodes and shards to contact for each query.
 ///
 /// When a query is prepared to be sent to ScyllaDB/Cassandra, a `LoadBalancingPolicy`
-/// implementation constructs a load balancing plan. That plan is a list of nodes to which
-/// the driver will try to send the query. The first elements of the plan are the nodes which are
+/// implementation constructs a load balancing plan. That plan is a list of
+/// targets (target is a node + an optional shard) to which
+/// the driver will try to send the query. The first elements of the plan are the targets which are
```

Comment:
nit: line width

Reply:
Not sure what you mean. This line has 100 characters, it doesn't seem excessive. There are also multiple other lines that exceed 80, so clearly it's not the limit we are using.

Reply:
OK, nvm then.